.changeset/fresh-comics-thank.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/code': minor
+---
+
+Adds `expandable` prop to `Code` component
```

Comment:
Can you add more details on what the expandable prop does and link the JIRA ticket? [Here's an example of that.](https://github.com/mongodb/leafygreen-ui/blob/main/packages/tabs/CHANGELOG.md#1300)